fission/src/systems/simulation/driver/HingeDriver.ts

```diff
@@ -84,20 +84,18 @@ class HingeDriver extends Driver {
 
         this.controlMode = DriverControlMode.Velocity
 
-        this._gravityChange = (event: PreferenceEvent) => {
-            if (event.prefName == "SubsystemGravity") {
-                const motorSettings = this._constraint.GetMotorSettings()
-                if (event.prefValue) {
-                    motorSettings.set_mMaxTorqueLimit(this._maxTorqueWithGrav)
-                    motorSettings.set_mMinTorqueLimit(-this._maxTorqueWithGrav)
-                } else {
-                    motorSettings.set_mMaxTorqueLimit(MAX_TORQUE_WITHOUT_GRAV)
-                    motorSettings.set_mMinTorqueLimit(-MAX_TORQUE_WITHOUT_GRAV)
-                }
+        this._gravityChange = (event: PreferenceEvent<"SubsystemGravity">) => {
+            const motorSettings = this._constraint.GetMotorSettings()
+            if (event.prefValue) {
+                motorSettings.set_mMaxTorqueLimit(this._maxTorqueWithGrav)
+                motorSettings.set_mMinTorqueLimit(-this._maxTorqueWithGrav)
+            } else {
+                motorSettings.set_mMaxTorqueLimit(MAX_TORQUE_WITHOUT_GRAV)
+                motorSettings.set_mMinTorqueLimit(-MAX_TORQUE_WITHOUT_GRAV)
             }
```

Comment:
If you're calling the same functions in both branches, it's probably better to just create max and min torque limit variables and assign them with a ternary. I know you didn't write this code, but I think it's best to conform it to our standard while we're at it.

```suggestion
            const [maxTorque, minTorque] = event.prefValue
                ? [this._maxTorqueWithGrav, -this._maxTorqueWithGray]
                : [MAX_TORQUE_WITHOUT_GRAV, -MAX_TORQUE_WITHOUT_GRAV]
            motorSettings.set_mMaxTorqueLimit(maxTorque)
            motorSettings.set_mMinTorqueLimit(minTorque)
```

Reply:
This code wasn't actually modified by this PR, just the function around it. Do you want me to change it anyway?

Reply:
I think this is a situation where I prefer the if statement over the ternary, since it makes it a lot more clear at a glance what the code is doing. I generally agree with the if -> ternary conversion when assigning as long as the condition and values aren't too long, but when assigning multiple values I think the regular if makes more sense.

EDIT: I suppose two separate ternaries could work as well, but I think the ternary with the unpacking takes longer to interpret than either of the other ways.

Reply:
I think the reason ternaries would work here is because the values intrinsically go together and are obviously bound by the same variable (min/max torque with or without gravity).